be/src/pipeline/exec/operator.h

```diff
@@ -157,11 +157,11 @@ class OperatorBase {
     explicit OperatorBase(OperatorBuilderBase* operator_builder);
     virtual ~OperatorBase() = default;
 
-    std::string get_name() const { return _operator_builder->get_name(); }
+    [[nodiscard]] std::string get_name() const { return _operator_builder->get_name(); }
 
-    bool is_sink() const;
+    [[nodiscard]] bool is_sink() const;
```

Comment:
only Status method_name() need nodiscard because we want the developer to check the ERROR STATUS.
But if the return value is string, for example, get_name, it not need to check the value.